docs/capped-bandwidth.md

```diff
@@ -2,63 +2,42 @@
-1. A small `epoch` size (eg `1 seconds`) would allow a membership to publish `24*3600/1=86400` messages a day, which would be too much. In exchange, this allows a user to publish messages right after the other, since it just have to wait 1 second between messages. Problem is that having an rln membership being able to publish this amount of messages, is a bit of a liability for waku, and hinders scalability.
-2. A high `epoch` size (eg `240 seconds`) would allow a membership to publish `24*3600/240=360` messages a day, which is a more reasonable limit, but this won't allow a user to publish two messages one right after the other, meaning that if you publish a message, you have to way 240 seconds to publish the next one. Not practical, a no go.
+Putting this all together and assuming:
+* Messages are sent uniformly distributed.
+* All users totally consumes its rate-limit.
 
-But what if we widen the window size, and allow multiple messages within that window?
-
-## Solution
-
-In order to fix this, we need bigger windows sizes, to smooth out particular bursts. Its fine that a user publishes 20 messages in one second, as long as in a wider window it doesn't publish more than, lets say 500. This wider window could be a day. So we could say that a membership can publish `250 msg/day`. With this we solve i) and ii) from the previous section.
-
-Some quick napkin math on how this can scale:
-- 10.000 RLN memberships
-- Each RLN membership allow to publish 250 msg/day
-- Message size of 5 kBytes
-
-Assuming a completely random distribution:
-- 10.000 * 250 = 2 500 000 messages will be published a day (at max)
-- A day has 86 400 seconds. So with a random distribution we can say that 30 msg/sec (at max)
-- 30 msg/sec * 5 kBytes/msg = 150 kBytes/sec (at max)
-- Assuming D_out=8: 150 kBytes/sec * 8 = 1.2 MBytes/sec (9.6 Mbits/sec)
-
-So while its still not possible to guarantee 100% the maximum bandwidth, if we rate limit per day we can have better guarantees. Looking at these numbers, considering a single shard, it would be feasible to serve 10.000 users considering a usage of 250 msg/day.
-
-TODO: Analysis on 95%/99% interval confidence on bandwidth given a random distribution.
-
-## TLDR
-
-- Waku should guarantee a capped bandwidth so that everyone can run a node.
-- The guarantee is a "statistical guarantee", since there is no way of enforcing a strict limit.
-- Current RLN approach is to rate limit 1 message every x seconds. A better approach would be x messages every day, which helps achieving such bandwidth limit.
-- To follow up: Variable RLN memberships. Eg. allow to chose tier 1 (100msg/day) tier 2 (200msg/day) etc.
\ No newline at end of file
+We can expect:
+* A traffic of `266 msg/second` on average (`160000/600`)
+* A traffic of `6 MBps` on average (266 * 4KB * 6), where `4KB` is the average message size and `6` is the average gossipsub D-out degree.
```

Comment:
Can you specify that it is per shard?

Reply:
fixed: af2101e657cfc6a336f992d6d0f38c873966a042